Maquetado/scss/styles.scss

```diff
@@ -2,6 +2,14 @@
   font-family: 'Roboto', sans-serif;
 }
 
-.body-container {
-  padding: $nav-height+50px 100px 0 100px;
+ @media screen and (min-width: 1024px) {
+  .body-container {
```

Comment:
this should be the default, remove the media query in line 5 and just use the media query in line 11 to override the styles for smaller resolutions